src/aura/disbursementsCreate/disbursementsCreate.css

```diff
@@ -0,0 +1,21 @@
+.THIS {}
+
+.THIS.wrap {
+    position: relative;
+    display: block;
+    position: relative;
+    width: 100%;
+    min-width: 550px;
+}
+
+.THIS .slds-item_label {
+    width: 50%;
+}
+
+.THIS .slds-item_detail {
+    width: 50%;
+}
+
+.THIS .inline_block {
```

Comment:
See comment on component about adding `slds-grid` to possibly eliminate this class.